backend/activity_log/api_views.py

```diff
@@ -55,9 +54,24 @@ class EventTypeViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.EventTypeSerializer
     queryset = models.EventType.objects.order_by("name")
     filter_backends = [DjangoFilterBackend]
-
+    filterset_fields = ["timestamp"]
```

Comment:
I'm not sure that this is fit for purpose. Filtering by an exact timestamp won't be very useful. The timestamps will be close to unique. 

Can you rather filter by timestamp_gte and timestamp_lte? 

timestamp_gte means timestamp greater than or equal to 
timestamp_lte means timestamp less than or equal to

Reply:
Please:
```
filterset_fields = ["timestamp_gte", "timestamp_lte"]
```
We need to filter by time stamp less than or equal to.
We also need to filter by timestamp greater than or equal to.